style.css

```diff
@@ -0,0 +1,90 @@
+body {
+    display: flex;
+   background-color:darkblue;
+   color: white;
+   justify-content: center;
+   align-items: center;
+   flex-direction: column;
+
+}
+#title {
+    text-align: center;
+    font-size: 25px;
+
+}
```

Comment:
Please ensure you always include an explanation, a "why", for each requested change. For example:
* This is a well-explained request: *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*